src/xunit.performance.api/ETWProfiler.cs

```diff
@@ -36,7 +35,8 @@ private static SafeTerminateHandler<TraceEventSession> MakeSafeTerminateTraceEve
         /// <returns></returns>
         public static void Record(string assemblyFileName, string runId, string outputDirectory, Action action, Action<string> collectOutputFilesCallback)
         {
-            if (!IsRunningAsAdministrator)
+            bool? isElevated = TraceEventSession.IsElevated();
+            if (!isElevated.HasValue || isElevated.Value == false)
```

Comment:
I don't think we should try to support any lower OS versions than PerfView does.  That would likely just be painful.